run/test/specs/message_checks_groups_ios.spec.ts

```diff
@@ -0,0 +1,507 @@
+import { iosIt } from "../../types/sessionIt";
+import { sleepFor, clickOnXAndYCoordinates } from "./utils";
+import { newUser } from "./utils/create_account";
+import { createGroup } from "./utils/create_group";
+import {
+  SupportedPlatformsType,
+  openAppThreeDevices,
+  closeApp,
+} from "./utils/open_app";
+import { runScriptAndLog } from "./utils/utilities";
+
+async function sendImageGroup(platform: SupportedPlatformsType) {
+  const testGroupName = "Message checks for groups";
+  const ronSwansonBirthday = "196705060700.00";
+  const { device1, device2, device3 } = await openAppThreeDevices(platform);
+  // Create users A, B and C
+  const [userA, userB, userC] = await Promise.all([
+    newUser(device1, "Alice", platform),
+    newUser(device2, "Bob", platform),
+    newUser(device3, "Charlie", platform),
+  ]);
+
+  // Create contact between User A and User B
+  await createGroup(
+    platform,
+    device1,
+    userA,
+    device2,
+    userB,
+    device3,
+    userC,
+    testGroupName
+  );
+  const testMessage = "Ron Swanson doesn't like birthdays";
+  const replyMessage = `Replying to image from ${userA.userName} in ${testGroupName}`;
+  await device1.clickOnElement("Attachments button");
+  await sleepFor(100);
+  await clickOnXAndYCoordinates(device1, 34, 498);
```

Comment:
maybe this function should allow some coordinates only, and have a meaning for it.
like await clickOnCoordinates(device1, Location.AttachmentsButton) or idk

Reply:
done